analyzers/tests/SonarAnalyzer.UnitTest/Common/SecurityHotspotTest.cs

```diff
@@ -67,7 +67,7 @@ private static string GetTestCaseFileName(DiagnosticAnalyzer analyzer)
 
             return typeName switch
             {
-                "ConfiguringLoggers" => "ConfiguringLoggers_AspNetCore",
+                "ConfiguringLoggers" => "ConfiguringLoggers_Log4Net",
```

Comment:
We only need a test suite to check if hotspot rule is disabled by default. `ConfiguringLoggers_AspNetCore` makes sense only on .Net or .Net Core and I prefer to use `ConfiguringLoggers_Log4Net` instead since it works for all runtimes.